tests/manage/z_cluster/test_must_gather.py

```diff
@@ -70,10 +76,40 @@ def finalizer():
 
         request.addfinalizer(finalizer)
 
+    @tier1
+    @pytest.mark.polarion_id("OCS-1583")
     def test_must_gather(self):
         """
         Tests functionality of: oc adm must-gather
 
+        """
+        self.validate_must_gather_content()
+
+    @tier4a
+    @pytest.mark.polarion_id("OCS-2328")
+    def test_must_gather_worker_node_down(self, nodes):
+        """
+        Collect must-gather OCS logs when a worker node is down
+
+        """
+        self.reboot_node = True
+        logger.info('Get all worker nodes and choose random from worker nodes list')
+        worker_nodes = get_worker_nodes()
+        worker_node = worker_nodes[randint(0, len(worker_nodes) - 1)]
+
+        # Stop one worker node
+        nodes.stop_nodes(get_node_objs([worker_node]))
+
+        # Collect must gather and check content
+        self.collect_must_gather()
```

Comment:
must-gather pod will fail to come if the pod tries to come on the down node.
We should pass --node-name in this case where one or more worker nodes are not in Ready state. 

 oc adm must-gather --image=registry.redhat.io/ocs4/ocs-must-gather-rhel8 --dest-dir=<directory-name> --node-name=<node-name>